pkg/cache/application_graphviz_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build graphviz
 // +build graphviz
```

Comment:
Do we need the old build tag? It will be removed in 1.18 according to https://www.jetbrains.com/help/go/configuring-build-constraints-and-vendoring.html#build-constraints.

I think you can remove the old tag.